src/goPath.ts

```diff
@@ -41,6 +41,15 @@ export function getBinPathWithPreferredGopath(toolName: string, preferredGopaths
 	}
 
 	const binname = alternateTool && !path.isAbsolute(alternateTool) ? alternateTool : toolName;
+	const env = Object.assign({}, process.env);
+	if (env['GOBIN'] != undefined){
+		const path = getBinPathFromEnvVar(binname, env['GOBIN'], false);
```

Comment:
Please rename to `pathFromGoBin` to match the patterns of `pathFromGoRoot` and `pathFromPath` names used later on